### PR TITLE
add jexiftoolGUI

### PR DESCRIPTION
**Pull request by muelli**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

I wanted to edit image metadata the other day and found that the current apps on flathub are a bit too heavy-handed, so I thought I'd package a simple GUI around exiftool.

I thought I have copied everything from the other Java app I maintain (josm) but it seems something still doesn't work as expected. By opening this PR, I hope that somebody more knowledgable than me comes along and tells me what to fix.

### CODE REVIEW

File: io.github.hvdwofl.jExifToolGUI.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+   <caption>Extra menus</caption>
+  </screenshot>
+  <screenshot type="default">
+   <image>https://hvdwolf.github.io/jExifToolGUI/screenshots/LinuxGeotagging-x600.jpg</image>
+   <caption>Geotagging</caption>
+  </screenshot>
+ </screenshots>
+ <url type="homepage">https://hvdwolf.github.io/jExifToolGUI/</url>
+ <update_contact>tobiasmue@gnome.org</update_contact>
+</component>
```
Comment (hfiguiere) [inline io.github.hvdwofl.jExifToolGUI.appdata.xml:34]: the appdata is missing the oars rating.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by nedrichards** [comment]
bot, build io.github.hvdwofl.jExifToolGUI

**Comment by barthalion** [comment]
bot, build io.github.hvdwofl.jExifToolGUI

**Comment by muelli** [comment]
umm.
there has been activity after the bot added the label. Who is responsible for this bot to add the logic?
How can I reopen this?
Let me ping @barthalion for not knowing what else to do.

**Comment by muelli** [comment]
sorry @barthalion for another ping. Maybe it's more sustainable to remove the stale flag.

**Comment by barthalion** [comment]
Oh yeah, I don't know why the bot is so insistive.

**Comment by muelli** [comment]
bot, build io.github.hvdwofl.jExifToolGUI

**Comment by barthalion** [comment]
It already notifies about a new release being available:

```
10:11:38.779 [AWT-EventQueue-0] INFO  org.hvdw.jexiftoolgui.Utils - Version on the web: 2.0.1
10:11:38.779 [AWT-EventQueue-0] INFO  org.hvdw.jexiftoolgui.Utils - This version: 1.8.1
```

I recommend setting up f-e-d-c and if possible, disabling this message somehow.

**Comment by hvdwolf** [comment]
I am willing to cooperate on this flatpack build, but currently do not know what is expected from me.
I also see this already started on 28 July 2021, but I completely missed that.

What is now required from me?
Am I the one needing to build the flatpack or is that done automatically, or does it require some trigger to automatically build upon a github rlease from me?

**Comment by muelli** [comment]
> What is now required from me?

I would describe the actual efforts as pretty low. I would even go as far as saying that you don't have to do anything, if you don't want to.
Most things are set up to be automatic, so an occasional review of a new release being incorporated is required. And eventually users will show up with bugs in the app running in the flatpak container which may require some more mental effort to deal with.
Ultimately, it's your app that's distributed through flathub, so you should get the opportunity to weigh in.

**Pull request closed without merging**